sdk/cognitivelanguage/Azure.AI.Language.Conversations/api/Azure.AI.Language.Conversations.netstandard2.0.cs

```diff
@@ -263,1094 +24,69 @@ public enum ServiceVersion
-        public string Subcategory { get { throw null; } set { } }
-        public string Text { get { throw null; } set { } }
-    }
-    public partial class GeneratedError
-    {
-        public GeneratedError(Azure.AI.Language.Conversations.ErrorCode code, string message) { }
-        public System.Collections.Generic.IDictionary<string, object> AdditionalProperties { get { throw null; } }
-        public Azure.AI.Language.Conversations.ErrorCode Code { get { throw null; } set { } }
-        public System.Collections.Generic.IList<Azure.AI.Language.Conversations.Models.GeneratedError> Details { get { throw null; } }
-        public Azure.AI.Language.Conversations.InnerErrorModel Innererror { get { throw null; } set { } }
-        public string Message { get { throw null; } set { } }
-        public string Target { get { throw null; } set { } }
+        protected ConversationAnalysisProjectsClient() { }
+        public ConversationAnalysisProjectsClient(System.Uri endpoint, Azure.AzureKeyCredential credential) { }
+        public ConversationAnalysisProjectsClient(System.Uri endpoint, Azure.AzureKeyCredential credential, Azure.AI.Language.Conversations.ConversationAnalysisClientOptions options) { }
+        public virtual System.Uri Endpoint { get { throw null; } }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
+        public virtual Azure.Operation<System.BinaryData> CancelTrainingJob(Azure.WaitUntil waitUntil, string projectName, string jobId, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Operation<System.BinaryData>> CancelTrainingJobAsync(Azure.WaitUntil waitUntil, string projectName, string jobId, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response CreateProject(string projectName, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> CreateProjectAsync(string projectName, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Operation<System.BinaryData> DeleteDeployment(Azure.WaitUntil waitUntil, string projectName, string deploymentName, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Operation<System.BinaryData>> DeleteDeploymentAsync(Azure.WaitUntil waitUntil, string projectName, string deploymentName, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Operation<System.BinaryData> DeleteProject(Azure.WaitUntil waitUntil, string projectName, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Operation<System.BinaryData>> DeleteProjectAsync(Azure.WaitUntil waitUntil, string projectName, Azure.RequestContext context = null) { throw null; }
```

Comment:
Can you also confirm that this returns a value on LRO completion?

Reply:
Yes, it's a RELO.